devlib/target.py

```diff
@@ -279,6 +280,14 @@ def shutils(self):
             self._setup_shutils()
         return self._shutils
 
+    def is_running(self, process):
+        cmd = "pgrep {}"
+        try:
+            self.execute(cmd.format(process))
```

Comment:
Can use f-string here

Reply:
Marc previously asked me to use .format in WA so I'm sticking with the convention here, I think it was for backwards compatibility.

Reply:
For backward compat it's not needed anymore since [July 2022](https://github.com/ARM-software/devlib/pull/569), we use [    python_requires='>= 3.7',](https://github.com/ARM-software/devlib/blob/be988bb42b766574bb9bd9191fa3532a65091716/setup.py#L85) for devlib and f-string are from 3.6

In WA the situation is different, as it must work with the last release of devlib in PyPI which is older than that, but inside devlib we can use anything that 3.7 offers.